js/models/order/orderFulfillment/CryptoDelivery.js

```diff
@@ -20,6 +26,12 @@ export default class extends BaseModel {
       addError('transactionID', app.polyglot.t('orderFulfillmentModelErrors.provideTransactionId'));
     }
 
+    if (attrs.transactionID.length > this.max.transactionIDLength) {
```

Comment:
This should be an else to the block above it because it doesn't make sense to check the length if the attr isn't provided in the first place.

Reply:
Ah, yeah. Makes sense.